packages/terra-table/src/terra-dev-site/doc/example/HeaderCheckMarkCellExample.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Table from 'terra-table';
+
+const HeaderCheckMarkCellExample = () => (
+  <React.Fragment>
+    <Table
+      summaryId="example-check-cell"
+      summary="This table displaying a default check column."
+      rowStyle="toggle"
+      checkStyle="readOnly"
+      headerData={{
+        selectAllColumn: { checkLabel: 'Column Example Title' },
+        cells: [{ id: 'column-0', key: 'key-0', children: 'test' }],
```

Comment:
I'd add better descriptive text in there (that tells the consumers what type of Header Cell each of them is) instead of "test"? 

<img width="789" alt="Screen Shot 2019-12-10 at 4 42 31 PM" src="https://user-images.githubusercontent.com/10090859/70525021-6a2d6800-1b6c-11ea-86ce-2555e83716d7.png">

Reply:
Updated.